src/Response.php

```diff
@@ -204,6 +204,15 @@ public function writeHead($status = 200, array $headers = array())
             );
         }
 
+        // assign date header if no 'date' is given, use the current time where this code is running
+        if (!isset($lower['Date'])) {
+            // example: "Date: Tue, 15 Nov 1994 08:12:31 GMT"
+            $headers = array_merge(
+                array('Date' => date('D, d M Y H:i:s T')),
```

Comment:
Thank you @kelunik and @clue . I changed the behavior in the latest commit. Check it out.